### DISCUSSION TITLE
Datepicker with Svelte

### DISCUSSION THREAD

**Discussion by pashadee** [tailwindlabs/tailwindcss#10539] [Help]
Hi guys,

Been banging my head against the desk for a while here trying to figure out how to use the datepicker component with svelte. I've been using all other components without to my difficulty by simply attaching on:click .. etc event handlers so that's been working fine. Wit the datepicker however, I get the code imported and get an input that has the calendar icon and all looks good, except that I can't figure out how to get the actual calendar (picker portion) of it to pop up once the input is clicked... it's not very clear about what attribute the component needs to bind to in order to trigger that... I tried {show} {display} all I could think of... if anyone could give me a quick pointer in the right direction it would be quite appreciated!

Thanks!

**Comment by ocamachor** [comment]
Check these components to see if any give you ideas.!
https://github.com/themesberg/flowbite-svelte/tree/main/src/lib/datepicker
